diff --git a/Makefile b/Makefile
index 0341f15..6217120 100644
--- a/Makefile
+++ b/Makefile
@@ -45,11 +45,9 @@ test: $(tests_binary)
 	$^ --exclude=integration
 
 $(tests_binary): $(SOURCE_FILES) | $(BUILD_DIR)
-	$(GET_DEPENDENCIES_WITH)
 	$(PONYC) -o $(BUILD_DIR) --bin-name $(notdir $(tests_binary)) $(TEST_SRC_DIR)
 
 $(binary): $(SOURCE_FILES) | $(BUILD_DIR)
-	$(GET_DEPENDENCIES_WITH)
 	$(PONYC) -o $(BUILD_DIR) --bin-name $(notdir $(binary)) $(SRC_DIR)
 
 clean:
@@ -59,7 +57,6 @@ clean:
 
 $(docs_dir): $(SOURCE_FILES)
 	rm -rf $(docs_dir)
-	$(GET_DEPENDENCIES_WITH)
 	$(PONYC) --docs-public --pass=docs --output build $(SRC_DIR)
 
 docs: $(docs_dir)
